Doc/reference/lexical_analysis.rst

```diff
@@ -365,7 +365,7 @@ characters:
 
 ``_*``
    Not imported by ``from module import *``.  The special identifier ``_`` is used
-   in the interactive interpreter to store the result of the last evaluation; it is
+   in the interactive interpreter to store the result of the last non-``None`` evaluation; it is
```

Comment:
Could you re-flow this to avoid lines longer than 79 chars? Actually, the whole paragraph should be re-flowed.